config.example.yml

```diff
@@ -38,21 +38,22 @@ workload_distribution_pipeline:
     results_folder: "azureml://FOLDER_PATH_WHERE_STORE_WORKLOAD_FILES"
 
 inference_pipeline:
-  model_parameters:
-    img_size: 4000
-    save_txt: True
-    exist_ok: True
-    half: True
-    hide_labels: True
-    save_blurred_image: True
-    batch_size: 1
-  inputs:
-    root_folder: "azureml://FOLDER_PATH_TO_IMAGES_FOLDER"
-    files_to_blur: "azureml://FILE_WITH_RELATIVE_PATHS_OF_BLUR_IMAGES"
-    model: "azureml://NAMED_ASSET_URI_MODEL_WEIGHTS_FILE"
-  outputs:
-    yolo_yaml_file: "azureml://FOLDER_PATH_WHERE_STORE_YAML_FILE_USED_BY_YOLO"
-    results_path: "azureml://FOLDER_PATH_WHERE_STORE_BLURRED_IMAGES"
+  customers:
+    - name: customer1
+      container_root: "azureml://subscriptions/{subscription}/resourcegroups/{resourcegroup}/workspaces/{workspace}/datastores/{datastore_name}/paths/"
```

Comment:
It seems a bit hacky to me, shall we not create data assets with the name of the customer and just use that as container root? Since this is why data assets are made in AzureML.